SpringBootRestWS/src/main/java/com/example/Entity/Person.java

```diff
@@ -0,0 +1,93 @@
+package com.example.Entity;
+
+import javax.persistence.Column;
+import javax.persistence.Entity;
+import javax.persistence.GeneratedValue;
+import javax.persistence.GenerationType;
+import javax.persistence.Id;
+
+@Entity
+public class Person {
+
+	@Id
+	@GeneratedValue(strategy = GenerationType.AUTO)
+	@Column(name = "id")
+	private Long id;
+
+	public Person(Long id) {
+		this.id = id;
+	}
+
+	@Column(name = "name")
+	private String name;
+
+	@Column(name = "email", nullable = false, unique = true)
+	private String email;
+
+	@Column(name = "password")
+	private String password;
+
+	@Column(name = "mobile")
+	private String PhoneNo;
```

Comment:
Incorect name of variable. PhoneNo - i think be better use full name, for example, phoneNumber. And name of variable we write with a small letter.